src/Microsoft.OData.Core/Microsoft.OData.Core.csproj

```diff
@@ -82,4 +82,8 @@
     </Compile>
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>
+

```

Comment:
what is this?

Reply:
It seems it's VS autogenerated.

---
In reply to: [518468018](https://github.com/OData/odata.net/pull/1913#discussion_r518468018) [](ancestors = 518468018)

Reply:
Can we remove it?

---
In reply to: [518470007](https://github.com/OData/odata.net/pull/1913#discussion_r518470007) [](ancestors = 518470007,518468018)

Reply:
Yes, it's removed